src/components/page-container/test/PageContainer.test.jsx

```diff
@@ -64,4 +65,38 @@ describe('PageContainer', () => {
       expect(await screen.findByText('Test Text')).toBeTruthy();
     });
   });
+
+  test('Page container with BackToInstructor compo without PUBLIC PATH', async () => {
+    await act(async () => {
+      const cohorts = { cohorts: [Factory.build('cohort'), Factory.build('cohort')] };
+      const courseMetadata = Factory.build('courseMetadata');
+
+      getCohorts.mockImplementation(() => cohorts);
+      getCourseHomeCourseMetadata.mockImplementation(() => courseMetadata);
+
+      render(
+        <PageContainer>
+          <BackToInstructor />
+        </PageContainer>,
+      );
+      const linkEl = await screen.findByText('Back to Instructor Dashboard');
+      expect(linkEl.href).toEqual('http://localhost:18000/courses/test-course-id/instructor#view-course-info');
+    });
+  });
+  test('Page container with BackToInstructor compo with PUBLIC_PATH', async () => {
```

Comment:
Instead of repeating tests that are similar except for a few variables (in this case, `PUBLIC_PATH`), I suggest you use `test.each()`.  Like [this instance in frontend-app-discussions](https://github.com/openedx/frontend-app-discussions/blob/master/src/discussions/discussions-home/BlackoutInformationBanner.test.jsx#L58-L63), for example.

Otherwise, this looks like a good place to include it!